src/rpc/blockchain.cpp

```diff
@@ -781,7 +781,8 @@ static RPCHelpMan getblockfrompeer()
         "getblockfrompeer",
         "\nAttempt to fetch block from a given peer.\n"
         "\nWe must have the header for this block, e.g. using submitheader.\n"
-        "\nReturns {} if a block-request was successfully scheduled\n",
+        "Calling this again for the same block and a new peer, will cause the response from the previous peer to be ignored.\n"
+        "\nReturns {} if a block-request was successfully scheduled.\n",
```

Comment:
4c5505be1438a8e0473adc7c96b5f681a9f81b02 maybe
```suggestion
        "\nReturns an empty JSON object if a block-request was successfully scheduled.\n",
```